src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -6,23 +6,27 @@
 import java.util.Properties;
 
 public class ConnectionUtil {
-    private static final String URL = "YOUR DATABASE URL";
-    private static final String USERNAME = "YOUR USERNAME";
-    private static final String PASSWORD = "YOUR PASSWORD";
-    private static final String JDBC_DRIVER = "YOUR DRIVER";
+    private static final String JDBC_DRIVER = "com.mysql.cj.jdbc.Driver";
+    private static final String URL = "jdbc:mysql://localhost:3306/taxi_db";
+    private static final String USERNAME = "root";
+    private static final String PASSWORD = "dsdfn";
 
     static {
         try {
             Class.forName(JDBC_DRIVER);
         } catch (ClassNotFoundException e) {
-            throw new RuntimeException("Can't find SQL Driver", e);
+            throw new RuntimeException("Can't find Driver for SQL", e);
         }
     }
 
     public static Connection getConnection() throws SQLException {
```

Comment:
i think if you throws in this block you dont need to use try-catch. Cause an exception declared with throws must be handled in the calling method, which calls the method that raised the exception.

Reply:
I don't think so)